[1LP][RFR] Move manual Ansible Tower test cases

nachandr opened the pull request:
<!-- Make sure to prefix the PR Title with any one of the below options...

   [WIP] - Work in Progress
   [WIPTEST] - Work in Progress with PRT Testing
   [RFR] - Ready for 1st Review
   [1LP][WIP] - 1st Level Pass, Work in Progress
   [1LP][WIPTEST] - 1st Level Pass, Work in Progress with PRT Testing
   [1LP][RFR] - 1st Level Pass, Ready for 2nd Review
-->

## Purpose or Intent
1)Deleted few test cases that have already been automated.
2)Moved few tests cases out of cfme/tests/test_manual.py


### PRT Run
<!--
- It's an internal RH service and only runs against signed whitelisted commits.
- It runs against a single appliance.
- Need to provide specific pytest expression else default it will run smoke tests [-m smoke].
- DOUBLE CURLY BRACES REQUIRED. Examples are in single to not get picked

Some examples are:
- {pytest: cfme/tests/test_foo_file.py -v}
- {pytest: cfme/tests/test_foo_file.py -k "test_foo_1 or test_foo_2" -v}
- {pytest: cfme/tests/ -k "test_foo_1 or test_foo_2 or test_foo_3" -v}
- {pytest: cfme/tests/test_foo_file.py --use-provider rhv43 -v}
- {pytest: cfme/tests/test_foo_file.py --long-running -v}

Notes:
- If PRT fails other than PR purpose/intent; please add a respective comment.
- For any guidance or assistance with PRT results; please contact to maintainers.
-->

Note for reviewers:
I'm looking into the Travis failure.

sshveta commented on cfme/tests/services/test_config_provider_servicecatalogs.py:
```diff
@@ -124,3 +124,74 @@ def test_retire_ansible_service(appliance, catalog_item, request, job_type):
+    5. Create new Catalog
+    6. Add new catalog item for Vsphere vcentre - VMWare
+    7. give it name, display in catalog, catalog,
+    8 Provisioning entry point:
```
dot is missing . (8.)

sshveta commented on cfme/tests/services/test_config_provider_servicecatalogs.py:
```diff
@@ -124,3 +124,74 @@ def test_retire_ansible_service(appliance, catalog_item, request, job_type):
+    deployed
+    - Vsphere template used for VM deployment must have ssh key "baked" in
+    -
+    Prompt for Extra variables must be enabled.
```
This can go in line 148

sshveta commented on cfme/tests/services/test_config_provider_servicecatalogs.py:
```diff
@@ -124,3 +124,74 @@ def test_retire_ansible_service(appliance, catalog_item, request, job_type):
+    /Service/Provisioning/StateMachines/ServiceProvision_Template/CatalogI
+    temInitialization
+    Request info tab:
+    Name of template: template_Fedora-Cloud-Base-23-vm-tools_v4 (this
```
These are sub points of Request Info tab right ? 
Can we start them with '-' as you did above in line 137 to 145

sshveta commented on cfme/tests/services/test_config_provider_servicecatalogs.py:
```diff
@@ -124,3 +124,74 @@ def test_retire_ansible_service(appliance, catalog_item, request, job_type):
+    14. Into Value column add:
+    /ConfigurationManagement/AnsibleTower/Operations/StateMachines/Job/def
+    ault?job_template_name=third_job_template
+    15. Automate -> Customization -> Service dialogs -> tower_dialog ->
```
Extra arrow at  the end

sshveta commented on cfme/tests/services/test_config_provider_servicecatalogs.py:
```diff
@@ -124,3 +124,74 @@ def test_retire_ansible_service(appliance, catalog_item, request, job_type):
+    param_prefix, otherwised will be ignored!!!
+    16. Order service
+    Into limit field put exact name of your VM:  test_tower_pakotvan_1234
+    17. Login into provision VM and `cat
```
Login into provisioned VM

nachandr commented:
@sshveta , Most of it was an oversight since I was just copying the existing tests from ./cfme/tests/test_manual.py :). I've addressed all your comments.
